lib/matplotlib/legend_handler.py

```diff
@@ -441,6 +451,7 @@ class HandlerErrorbar(HandlerLine2D):
     """
     Handler for Errorbars.
     """
+
```

Comment:
Were all these extra carriage returns called for somehow?

Reply:
I believe the pep8 rules that I followed required a blank line after dosctrings in methods, correct me if i'm wrong please.